benchmarks/src/bench/find_many.rs

```diff
@@ -1,13 +1,11 @@
-use std::{fs::File, path::PathBuf};
+use std::{convert::TryInto, fs::File, path::PathBuf};
 
-use bson::Bson;
-use mongodb::{Client, Collection, Database};
+use anyhow::{bail, Result};
```

Comment:
Since our error types for BSON have changed a bit since last summer and I didn't think it was worth manually updating the boilerplate, I switched this to use [`anyhow`](https://crates.io/crates/anyhow), which is basically a wrapper around `Box<dyn Error>`. While this might be marginally less performant due to allocating for errors, the benchmarks shouldn't ever actually run into errors, and if they do, we don't really care about the performance results, so I think it's fine.